graviti/schema/builtin.py

```diff
@@ -258,6 +312,24 @@ def __getitem__(self, index: Union[int, str, slice]) -> Union[PortexType, List[P
 
         return self.fields[index].type
 
+    def _get_to_pyarrow_params(self) -> Dict[str, Any]:
+        return {"nullable": self.nullable}
+
+    def _pyarrow_type_factory(self) -> "paDataType":
+        return pa.struct(self.fields.to_pyarrow())  # pylint: disable=no-member
+
+    def to_pyarrow(self, _to_schema: bool = True) -> pa.Schema:
```

Comment:
do we need `overload` for this method?
if `_to_schema` is `False`, will this method return `pa.Schema`?